### PR TITLE
Milestone 2 : Add functionalities

### PR DESCRIPTION
**Pull request by Lizdev-05**
In this milestone, I achieved the following...

- Read the Leaderboard API documentation

- Create a new game with the name of your choice by using the API.

- Implement the "Refresh" button (receiving data from the API and parsing the JSON).

- Implement the form "Submit" button (sending data to the API).

- Use arrow functions instead of the function keyword.

- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/displayScore.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const displayScores = async () => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/eJRS7w29YVAaVnC0rjhT/scores');
+  const data = await response.json();
+  let dataArray = data.result;
+  dataArray = dataArray.sort((a, b) => b.score - a.score);
+  scoreList.innerHTML = '';
+  dataArray.forEach((score) => {
+    addScore(score.user, score.score);
+  });
```
Comment (BenMukebo) [inline src/displayScore.js:13]: - [OPTIONAL] Great job @Lizdev-05 👍🏻 what if you use the JavaScript try and catch methods ? it's really simple:

```
const getScores = async () => {
  try{
    let response = await fetch(https:// ....);
        const data = await response.json();
        let dataArray = data.result;
       dataArray = dataArray.sort((a, b) => b.score - a.score);
       scoreList.innerHTML = '';
       dataArray.forEach((score) => {
         addScore(score.user, score.score);
       });
  } catch(error){
    console.log(error.message)
  }
};
```
- For more documentation you can check  [JavaScript try and catch](https://www.w3schools.com/js/js_errors.asp)

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Lizdev-05,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ]  Please consider adding the correct information concerning your project in the README file. All the headers in the template need to be filled properly so that your README is descriptive and professional.

- [ ] Kindly, Add a descriptive README file to your project,  A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally as this will be of help to non-technical users.  this includes `git clone URL` , `cd project_name`, `npm install`, `npm start`, etc.
- Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project. 
- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._ser. 


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/displayScore.js (hunk 1 above)
Comment (Lizdev-05) [reply src/displayScore.js:13]: Thank you for the review. As regards the catch error, will I have to display the error message or I will just log it into the console?

Comment (BenMukebo) [reply src/displayScore.js:13]: @Lizdev-05  you don't really need to modify your code I just wanted to give you an idea about how you could test your code using Try and Catch. The try statement allows you to define a block of code to be tested for errors while it is being executed. The catch statement allows you to define a block of code to be executed if an error occurs in the try block. but according to the Microvese [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log%20to%20your%20repo.%20It%E2%80%99s%20ugly%2C%20it%20kills%20performance%20and%20it%20can%20make%20confidential%20data%20be%20visible%20to%20anyone%20using%20the%20browser%20tools%20to%20look%20at%20your%20website.), We do not commit `console.log` to the repo.

Comment (Lizdev-05) [reply src/displayScore.js:13]: @BenMukebo, I understand. Thank you so much for your guidance.

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Lizdev-05 , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**